app/controllers/users_controller.rb

```diff
@@ -1,10 +1,9 @@
 class UsersController < ApplicationController
   before_action :verify_user, only: [:edit, :update]
+
   def index
-    @users = Array.new
-    User.all[0..5].each do |user|
-      @users << user
-    end
+    # This gets the first 5 Users, in order of ascending id
+    @users = User.find(:all, :order => "id asc", :limit => 5)
```

Comment:
I tried adding this line but my editor says that find(:all) has been deprecated in favor of all method.